spec/requests/api/collections_spec.rb

```diff
@@ -244,5 +244,11 @@ def test_collection_query(collection, collection_url, klass, attr = :id)
       FactoryGirl.create(:container_deployment)
       test_collection_query(:container_deployments, container_deployments_url, ContainerDeployment)
     end
+
+    it 'queries ArbitrationDefauls' do
+      ems = FactoryGirl.create(:ext_management_system)
+      FactoryGirl.create(:arbitration_default, :ems_id => ems.id)
```

Comment:
It might be better to do `:ems => ems` here rather than set the id explicitly. I think the latter might skip callbacks, so your `ems` won't know anything about the relationship